content-src/components/DiscoveryStreamComponents/DSContextFooter/DSContextFooter.jsx

```diff
@@ -9,32 +9,36 @@ import React from "react";
 // Animation time is mirrored in DSContextFooter.scss
 const ANIMATION_DURATION = 3000;
 
-const StatusMessage = ({ icon, fluentID }) => (
+export const StatusMessage = ({ icon, fluentID }) => (
   <div className="status-message">
     <span
       aria-haspopup="true"
       className={`story-badge-icon icon icon-${icon}`}
     />
-    <div className="story-context-label" data-l10n-id={fluentID} />
+    <div className="story-context-label clamp" data-l10n-id={fluentID} />
   </div>
 );
 
 export class DSContextFooter extends React.PureComponent {
   render() {
-    const { context, context_type } = this.props;
+    const { context, context_type, engagement = "Fresh" } = this.props;
```

Comment:
If we're expecting to turn on for de, we should get these exposed to localizers soon